src/zenml/config/global_config.py

```diff
@@ -701,6 +712,30 @@ def set_store(
         self._configure_store(config, skip_default_registrations, **kwargs)
         logger.info("Updated the global store configuration.")
 
+    def close_store(self) -> None:
+        """Close and detach the cached store if it has been initialized.
+
+        Important:
+        - Operates only on the cached store; does not instantiate a new one.
+        - Logs cleanup errors at debug level and never raises.
+        """
+        store = self._zen_store
+        if store is None:
+            return
+
+        close_fn = getattr(store, "close", None)
+        if callable(close_fn):
+            try:
+                close_fn()
+            except Exception as e:
+                logger.debug(
+                    "Error while closing zen store during cleanup: %s",
+                    e,
+                    exc_info=True,
+                )
+        # Clear the cached store reference irrespective of cleanup result.
```

Comment:
There's no real reason to make this code so ugly:
If we want a close (or maybe `disconnect`?) method, it should be defined on the `ZenStoreInterface`, which means you can simply call `store.close()` here. Additionally, you'll also be able to implement that functionality for the Rest store, which can potentially also drain it's connections. Or it simply does nothing.

Reply:
Done.